lib/routes/api/accounts.js

```diff
@@ -1038,4 +1041,212 @@ router.get('/:sid/Queues', async(req, res) => {
   }
 });
 
+// Tts
```

Comment:
accounts.js is getting rather big.  Can you please factor all this new code into a new file